projects/adt7420-pmdz/src/examples/dummy/dummy_example.c

```diff
@@ -85,6 +85,8 @@ int dummy_example_main()
 	if (ret)
 		goto error_adt7420;
 
+	no_os_mdelay(240);
```

Comment:
nit: usually for these specific delay values, if they are datasheet related, we add a comment in the code.